spec/controllers/requests_controller_spec.rb

```diff
@@ -99,9 +99,9 @@
         end.to change(Request, :count).by(1)
       end
 
-      it 'redirects to the created request' do
+      it 'redirects to the request overview if user is not an admin' do
         post :create, params: { request: valid_attributes }
-        expect(response).to redirect_to(Request.last)
+        expect(response).to redirect_to(requests_url)
       end
```

Comment:
@MrModder is there an elegant way to create a second test, that tests the redirection for an admin user?
I am not sure how to do this, since for this file the signed in user seems to always be wimi.

Reply:
You somehow have to create an admin user (should be possible by adapting the user factory code)
Then you can just use:
``` ruby
user = FactoryBot create :admin
sign_in(user) #this will sign in the user via devise, do this before you request anything
```
You can take a look at ```slack_controller_spec.rb``` the ```mock_sign_in``` function does this with a standard user.